docs/architecture/cloud-native/host-eshoponcontainers-application.md

```diff
@@ -1,27 +1,42 @@
+
+Once the cluster has been created and configured, you can deploy the application to it using Helm and Tiller.
 
 ## Helm Deployment
 
-asdf
+Basic deployments to AKS may use custom CLI scripts or simple deployment files, but more complex applications should use a dependency management tool like Helm. Helm is maintained by the Cloud-native Computing Foundation and helps you define, install, and upgrade Kubernetes applications. Helm is composed of a command line client, helm, which uses helm charts, and an in-cluster component, Tiller. Helm Charts use standard YAML-formatted files to describe a related set of Kubernetes resources, and are typically versioned alongside the application they describe. Helm Charts range from extremely simple to very complex depending on the requirements of the installation they describe.
+
+You'll find the eShopOnContainers helm charts in the /k8s/helm folder. Figure 3-X shows how the different components of the application are organized into a folder structure used by helm to define and managed deployments.
+
+![eShopOnContainers Architecture](media/eshoponcontainers-helm-folder.png)
+**Figure 3-X**. The eShopOnContainers helm folder.
+
+Each individual component is installed using a `helm install` command. These commands is easily scripted, and eShopOnContainers provides a "deploy all" script that loops through the different components and installs them using their respective helm charts. The result is a repeatable process, versioned with the application in source control, that anyone on the team can deploy to an AKS cluster with a one line script command. Especially when combined with DevSpaces, this makes it very easy for developers to diagnose and test their individual changes to their microservice-based cloud-native apps.
 
 ## DevSpaces for Kubernetes
 
-asdf
+Azure Dev Spaces helps individual developers host their own unique version of AKS clusters in Azure during development. This minimize local machine requirements and allows team members to quickly see how their changes will behave in a real AKS environment. Azure Dev Spaces offers a CLI for developers to use to manage their dev spaces and to deploy to a specific child dev space as needed. Each child dev space is referenced using a unique URL subdomain, allowing side-by-side deployments of modified clusters so that individual developers can avoid conflicting with each other's work in progress.
```

Comment:
Can you add the Image that explains how developers can use Azure Dev Spaces? We have to improve this explanation here since we don't have another section to explain this in detail.

Reply:
The one in the intro chapter was removed. It was too soon to talk about that in length. What section are you planning to have this covered?